### PR TITLE
AMP-20 Share bar under the main image on article page

### PR DESCRIPTION
**Pull request by dosyara**
Added share bar to an article.
Updated some styles to match main site.

### CODE REVIEW

File: src/app/components/styled-share-bar.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  padding: ${spacings.s} 0;
+
+  amp-social-share,
+  .comments-link {
+    background-color: ${color.kiev};
+    box-sizing: border-box;
+    width: 31px;
+    height: 31px;
+    border-radius: 9999999px;
+    fill: #fff;
```
Comment (yasgreen93) [inline src/app/components/styled-share-bar.js]: `${color.thimphu};`

File: src/app/components/styled-share-bar.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  border-style: solid;
+  border-color: ${color.cardiff};
+  padding: ${spacings.s} 0;
+
+  amp-social-share,
+  .comments-link {
+    background-color: ${color.kiev};
+    box-sizing: border-box;
+    width: 31px;
+    height: 31px;
```
Comment (yasgreen93) [inline src/app/components/styled-share-bar.js]: `31px` seems to be used a lot, could we put it in a const somewhere and reuse it?

File: src/app/components/article-description.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ const rubricClassName = css`
   font-size: ${text.sizeStep['0']};
   margin: 0 0 ${spacings.m};
   font-style: italic;
-  line-height: ${text.lineHeight.serifMediumOnStep['0']};
+  line-height: 1.4;
```
Comment (wajerrr) [inline src/app/components/article-description.js]: I assume that this change is due to mismatch with OnStep values ?

Comment (dosyara) [reply src/app/components/article-description.js]: Correct. Rubrics on a main site has line height 1.4, but we do not have this variable in typography.

**Comment by umbobabo** [comment]
Elegant PR. Merging.

**Pull request merged**